source/js/buyers-guide/bg-main.js

```diff
@@ -93,7 +93,7 @@ let main = {
     let envReq = new XMLHttpRequest();
 
     envReq.addEventListener(`load`, () => {
-      callback.call(this, JSON.parse(envReq.response));
+      callback(JSON.parse(envReq.response));
```

Comment:
Also, `JSON.parse` can throw, which in this case would mean that it would break the entire script load - if we're updating this code, let's add a few lines to harden this:
```
try {
  processEnv(JSON.parse(envReq.response));
} catch (e) {
  processEnv({});
}
```

Reply:
oh yea we need this!